Forms/DateOfBirthUtils.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using UnityEngine;
+
+namespace DUCK.Forms
+{
+	public static class DateOfBirthUtils
+	{
+		public static int[] Days { get; private set; }
+		public static int[] Years { get; private set; }
+		public static string[] Months { get; private set; }
+
+		public enum MonthType
+		{
+			January,
+			Febuary,
+			March,
+			April,
+			May,
+			June,
+			July,
+			August,
+			September,
+			October,
+			November,
+			December
+		}
+
+		static DateOfBirthUtils()
+		{
+			const int days = 31;
+			Days = new int[31];
+			for (var day = 0; day < days; day++)
+			{
+				Days[day] = day + 1;
```

Comment:
You could start the loop at 1, and have the condition `day <= days` instead of adding 1 each time

Reply:
Then the index would be out of range, I think the naming is confusing, I should change `day` to `i`